src/components/stepper/index.js

```diff
@@ -14,6 +14,10 @@ const Stepper = ({ header, children, onComplete }) => {
     }
 
     const onNextClick = () => {
+        if (data.firstName.length === 0 || data.lastName.length === 0) {
```

Comment:
This `Stepper` is a generic component that shouldn't care about the specific data being passed in to it. This line of code is tightly coupling it to the data, removing it's use as a general reusable component. You'll need to figure out a different approach to keep this component general use